ansible/files/steps/batch.sh

```diff
@@ -89,17 +89,17 @@ else
     fi
     if [ $EXTENSION = "json" ]; then
         # Executes Zeppelin notebook
-        ZEPPELIN_PATH=/mnt/anaconda2/envs/zeppelin
-        source activate zeppelin
-        ${ZEPPELIN_PATH}/bin/zeppelin-execute -i ../${NOTEBOOK_NAME} -o ./${NOTEBOOK_NAME}
+        ANACONDA_PATH={{telemetry_analysis_anaconda_path}}
+        source $ANACONDA_PATH/bin/activate zeppelin
+        $CONDA_PREFIX/bin/zeppelin-execute -i ../${NOTEBOOK_NAME} -o ./${NOTEBOOK_NAME}
```

Comment:
Wondering if we change the syntax to `${CONDA_PREFIX}` to match how `${NOTEBOOK_NAME}` is being called, or vice versa.